### DISCUSSION TITLE
Lucida fonts don't work

### DISCUSSION THREAD

**Discussion by danylog** [olikraus/u8g2#1648] [Q&A]
Hi, I just installed u8g2 and I needed some italic fonts to be displayed. So, I found them (for example "u8g2_font_lubBI14_te"), and tried to insert in the standard "u8g2.setFont", but they do not set as blue in my Arduino IDE, and if I try to compile, it says that it is undeclared(but the last font was declared) It happens to me with all "LUCIDA" fonts and they are the only which are in italic. I don't know what to do((( 
I'm using SSD1306 128*32 OLED via I2C 

Angelina)

**Comment by olikraus** [comment]
I just tried `u8g2_font_lubBI14_te` and it works as expected.

Which version of U8g2 did you install?

**Reply by danylog** [reply]
Uhm, I don't know, I think the last, let me check readme.
2.20.13

**Comment by olikraus** [comment]
Lucida fonts got included in 2.27: https://github.com/olikraus/u8g2/issues/271

You probably need to update u8g2.
